twitter/API/APIManager.h

```diff
@@ -18,4 +18,8 @@
 
 - (void)postStatusWithText:(NSString *)text completion:(void (^)(Tweet *, NSError *))completion;
 
+- (void)favorite:(Tweet *)tweet completion:(void (^)(Tweet *, NSError *))completion;
```

Comment:
In a lot of our header files we'll do some documentation and I've seen this in some of your other pull requests. You can see how we do some of that [here](https://fburl.com/wiki/vv5jxcdr). This is a good habit to have especially as you begin to work with more and more people.